pousse-cafe-shop-app/src/main/java/poussecafe/shop/domain/mongo/CustomerDataAccess.java

```diff
@@ -2,10 +2,17 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.mongodb.repository.MongoRepository;
+import poussecafe.discovery.DataAccessImplementation;
+import poussecafe.shop.domain.Customer;
 import poussecafe.shop.domain.CustomerId;
 import poussecafe.spring.mongo.storage.MongoDataAccess;
+import poussecafe.spring.mongo.storage.SpringMongoDbStorage;
 
-public class CustomerDataAccess extends MongoDataAccess<CustomerId, CustomerData, String> {
+@DataAccessImplementation(
+        aggregateRoot = Customer.class,
+        dataImplementation = CustomerData.class,
+        storageName = SpringMongoDbStorage.NAME
+)public class CustomerDataAccess extends MongoDataAccess<CustomerId, CustomerData, String> implements poussecafe.shop.domain.CustomerDataAccess<CustomerData> {
```

Comment:
Put the public class... on the next line

Reply:
Done.